hs_core/hydroshare/utils.py

```diff
@@ -661,10 +661,10 @@ def validate_user_quota(user_or_username, size):
                 QuotaMessage.objects.create()
             qmsg = QuotaMessage.objects.first()
             used_size = uq.add_to_used_value(size)
-            used_percent = uq.used_percent
-            rounded_percent = round(used_percent, 2)
-            rounded_used_val = round(used_size, 4)
-            if used_percent >= 100:
+            if used_size >= uq.allocated_value:
+                used_percent = used_size / uq.allocated_value * 100
+                rounded_percent = round(used_percent, 2)
+                rounded_used_val = round(used_size, 4)
```

Comment:
fix the quota validation

Reply:
Is there a unit test around this function? It is not critical because quota is ultimately enforced by MinIO but the test could ensure quota messaging is accurately reported to the user.